pl_examples/run_examples-args.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
-
-echo $@
-
-full_path=$(realpath $0)
-echo $full_path
-
-dir_path=$(dirname $full_path)
```

Comment:
If you drop this, you cannot call it from elsewhere and we need it for ngc testing

Reply:
Updated

Reply:
@Borda Mind checking this PR again ?